release.json

```diff
@@ -8,7 +8,8 @@
         "MACOS_BUILD_VERSION": "master",
         "WINDOWS_DDNPM_DRIVER": "release-signed",
         "WINDOWS_DDNPM_VERSION": "0.98.2.git.86.53d1ee4",
-        "WINDOWS_DDNPM_SHASUM": "5d31cbf7aea921edd5ba34baf074e496749265a80468b65a034d3796558a909e"
+        "WINDOWS_DDNPM_SHASUM": "5d31cbf7aea921edd5ba34baf074e496749265a80468b65a034d3796558a909e",
+        "SECURITY_AGENT_POLICIES_VERSION": "master"
```

Comment:
This should not be changed here

Reply:
I found that while testing the build types this PR changes. The absence of `SECURITY_AGENT_POLICIES_VERSION` in the `nightly` entry makes A6 nightly builds work only by accident (because the software definition that uses it defaults to `master` when the variable is not defined). I'd rather have it explicitly defined here (as it is for all other entries) to prevent mistakes in the future.

I changed it here alongside all other pipeline mistakes I found along the way, but I can put it in another PR if you prefer.

Reply:
We can keep it here but maybe add a note to the PR description with this for future reference

Reply:
Done